suite-native/module-settings/src/components/ViewOnly/WalletRow.tsx

```diff
@@ -42,6 +42,10 @@ export const WalletRow = ({ device }: { device: AcquiredDevice }) => {
             icon: 'check',
         });
         dispatch(toggleRememberDevice({ device }));
+
+        if (device.remember) {
+            dispatch(deviceActions.forgetDevice(device));
```

Comment:
should we forget device that is connected?

Reply:
definitely not 🙉 thanks for the catch

Reply:
fixed